bot/seasons/season.py

```diff
@@ -132,11 +134,18 @@ def greeting(self) -> str:
         """
         return f"New Season, {self.name_clean}!"
 
-    async def get_icon(self, avatar: bool = False) -> bytes:
+    @property
+    def cycle(self) -> bool:
+        """Return whether the icon attribute is a list and hence needs to be cycled through."""
+        return isinstance(self.icon, list)
+
+    async def get_icon(self, avatar: bool = False, index: int = 0) -> List[Union[bytes, str]]:
```

Comment:
The return type annotation is wrong because it's actually returning a tuple; use `typing.Tuple` instead. You don't need `Union` for it because you specify the type of each position in the tuple directly.